### PR TITLE
Remove empty link in user menu

### PR DESCRIPTION
**Pull request by Steph4104**
Add span with message for screen reader. 
Change the link to href.

fix issue #918

### CODE REVIEW

**Comment by Steph4104** [comment]
@piet0024  can you test with JAWS and tell me what I miss. I use a free screen reader and everything is fine for me now.

File: mod/wet4/views/default/page/elements/user_menu.php [hunk 1]
```diff
@@ -125,14 +125,16 @@
 //Nick - Removed the href and created my own in the text to hold hidden messages dropdown
 elgg_register_menu_item('user_menu', array(
     'name' => 'messages',
-    'text' => '<a href="'.elgg_get_site_url().'messages/inbox/' . $user.'"><i class="fa fa-envelope mrgn-rght-sm mrgn-tp-sm fa-lg"></i><span class="hidden-xs">' . elgg_echo('messages') . '</span>' . $msgbadge .'</a>'.$focus_dd .$ajax_dd_messages,
+    'text' => /*'<a href="'.elgg_get_site_url().'messages/inbox/' . $user.'">*/'<i class="fa fa-envelope mrgn-rght-sm mrgn-tp-sm fa-lg"></i><span class="hidden-xs" aria-hidden="true">' . elgg_echo('messages') . '</span>' . $msgbadge .'<span class="wb-inv">'.elgg_echo('userMenu:messages') . $title.' </span></a>'.$focus_dd .$ajax_dd_messages,
```
Comment (nschonni) [inline mod/wet4/views/default/page/elements/user_menu.php]: Leaving dead code is not usually helpfull

File: mod/wet4/views/default/page/elements/user_menu.php [hunk 2]
```diff
@@ -125,14 +125,16 @@
...
     'title' => elgg_echo('userMenu:messages') . $title,
     'item_class' => 'brdr-lft messagesLabel close-msg-dd',
     'data-toggle' => '',
     'id'=>'messagesLabel',
     'class' => '',
     'data-dd-type'=>'msg_dd',
     'priority' => '2',
+    'href' => elgg_get_site_url().'messages/inbox/' . $user,
+    //'tabindex' => '-1'
```
Comment (nschonni) [inline mod/wet4/views/default/page/elements/user_menu.php]: Leaving dead code is not usually helpfull

File: mod/wet4/views/default/page/elements/user_menu.php [hunk 3]
```diff
@@ -159,13 +161,13 @@
 // notifications inbox menu item
 elgg_register_menu_item('user_menu', array(
     'name' => 'notifications',
-    'text' => '<a href="'.elgg_get_site_url().'messages/notifications/' . $user.'"><i class="fa fa-bell mrgn-rght-sm mrgn-tp-sm fa-lg"></i><span class="hidden-xs">' . elgg_echo('notifications:subscriptions:changesettings') . '</span>' . $msgbadge .'</a>'.$focus_dd.'<div>'.$ajax_dd_notification.'</div>',
+    'text' => /*'<a href="'.elgg_get_site_url().'messages/notifications/' . $user.'">*/'<i class="fa fa-bell mrgn-rght-sm mrgn-tp-sm fa-lg"></i><span class="hidden-xs">' . elgg_echo('notifications:subscriptions:changesettings') . '</span>' . $msgbadge .'<span class="wb-inv">'.elgg_echo('userMenu:notifications') . $title.' </span></a>'.$focus_dd.'<div>'.$ajax_dd_notification.'</div>',
```
Comment (nschonni) [inline mod/wet4/views/default/page/elements/user_menu.php]: Leaving dead code is not usually helpfull

**Pull request merged**